### PR TITLE
[Feature Store] Add docstrings

### PR DESCRIPTION
**Pull request by katyakats**

### CODE REVIEW

File: mlrun/datastore/sources.py [hunk 1]
```diff
@@ -137,6 +137,24 @@ def to_dataframe(self):
...
 
 class ParquetSource(BaseSourceDriver):
+    """
+       Reads Parquet file as input source for a flow.
+
+       :parameter name: name of the source
+       :parameter path: path to Parquet file or directory
+       :parameter key_field: the column to be used as the key for events. Can be a list of keys.
+       :parameter time_field: the column to be parsed as the timestamp for events. Defaults to None
+            (no timestamp field).
```
Comment (dinal) [inline mlrun/datastore/sources.py:147]: remove

File: mlrun/datastore/sources.py [hunk 2]
```diff
@@ -137,6 +137,24 @@ def to_dataframe(self):
...
+       :parameter start_filter: datetime. If not None, the results will be filtered by partitions and
+            'filter_column' >= start_filter. Default is None
+       :parameter end_filter: datetime. If not None, the results will be filtered by partitions
+            'filter_column' < end_filter. Default is None
+       :parameter filter_column: Optional. if not None, the results will be filtered by this column and before and/or
+            after
```
Comment (dinal) [inline mlrun/datastore/sources.py:153]: fix

File: mlrun/datastore/sources.py [hunk 3]
```diff
@@ -137,6 +137,24 @@ def to_dataframe(self):
...
+       :parameter schedule: string to configure scheduling of the ingestion job.
```
Comment (dinal) [inline mlrun/datastore/sources.py:154]: maybe add some example on how the string should look like

**Review by dinal — APPROVED** [review]

File: mlrun/datastore/sources.py [hunk 4]
```diff
@@ -137,6 +137,24 @@ def to_dataframe(self):
 
 
 class ParquetSource(BaseSourceDriver):
+    """
+       Reads Parquet file as input source for a flow.
```
Comment (Hedingber) [inline mlrun/datastore/sources.py:141]: Note you're saying parquet **file** while it can be either file or directory

**Review by Hedingber — CHANGES_REQUESTED** [review]

**Review by Hedingber — APPROVED** [review]

**Pull request merged**